### PR TITLE
Updated create namespace to handle retry logic

### PR DESCRIPTION
**Pull request by dislbenn**
Related issue: https://github.com/open-cluster-management/backlog/issues/17503
Signed-off-by: dislbenn <lavontae.bennett@gmail.com>

### CODE REVIEW

File: COMPONENT_VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.4.0
+2.5.0
```
Comment (mqhackett) [inline COMPONENT_VERSION:1]: Out of curiosity, will these test continue to run for previous releases in any capacity?

Comment (dislbenn) [reply COMPONENT_VERSION:1]: I believe so, this is just updating the component version that we're using for the main branch (which should be pointing towards 2.5). On the other release branches, the component version should remain consistent with the release that they are building off of.

**Review by mqhackett — APPROVED** [review]

**Comment by mqhackett** [comment]
Looks good, once the e2e passes

**Comment by dislbenn** [comment]
Force merging this PR. The `ci/prow/e2e` still needs to be debugged and implemented properly within this repo. We will debug the prow job within a separate PR: https://github.com/open-cluster-management/search-e2e-test/pull/169.

**Pull request merged**